From f2326e823c63264e6e8c42cb65e1ee90c896c570 Mon Sep 17 00:00:00 2001
From: "Pham Thanh Tam [FGA.AIS]" <tampt8@fsoft.com.vn>
Date: Tue, 23 Jul 2019 08:29:33 +0700
Subject: [PATCH 225/628] ARM: DTS: r8a77470: Add DMAC devices to DT

Instantiate the two system DMA controllers in the r8a77470 device tree.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Pham Thanh Tam [FGA.AIS] <tampt8@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a77470.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index bcd9c96..3d7bfd2 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -289,7 +289,7 @@
 					  "ch12", "ch13", "ch14";
 			clocks = <&mstp2_clks R8A77470_CLK_SYS_DMAC0>;
 			clock-names = "fck";
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			#dma-cells = <1>;
 			dma-channels = <15>;
 		};
@@ -321,7 +321,7 @@
 					  "ch12", "ch13", "ch14";
 			clocks = <&mstp2_clks R8A77470_CLK_SYS_DMAC1>;
 			clock-names = "fck";
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			#dma-cells = <1>;
 			dma-channels = <15>;
 		};
-- 
2.7.4

